index.js

```diff
@@ -78,9 +82,23 @@ AudioManager.prototype.init = function () {
 };
 
 //▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
-/** Get a unused sound object (or a new one if no more empty sounds are available in pool) */
-AudioManager.prototype.getEmptySound = function () {
+/** Get a unused sound object (or a new one if no more empty sounds are available in pool)
+ *
+ * @param {string} channelId - channel id where the sound will be used
+ * @param {string} soundId   - sound id
+ *
+ * @return {Object} sound object
+ */
+AudioManager.prototype.getEmptySound = function (channelId, soundId) {
 	var sound;
+
+	// custom sound constructor
+	var constructor = this.settings.getSoundConstructor(channelId, id);
+	if (constructor) {
+		sound = new constructor();
```

Comment:
My comment has been invalidated by the latest PR:
https://github.com/Wizcorp/AudioManager/pull/23/commits/f9f66594148b3833d121f12e595968951c43ea36
;)